README.md

```diff
@@ -151,13 +151,15 @@ build to keep the Docker image up-to-date.
 
 [docker-base]: https://github.com/nextstrain/docker-base
 
+
+## Documentation
+
 ### Building documentation
 
-[Documentation](https://nextstrain-augur.readthedocs.io) is built using [Sphinx](http://sphinx-doc.org/) and hosted on [Read The Docs](https://readthedocs.org/).
-Versions of the documentation for each augur release and git branch are available and preserved.
+[Documentation] is built using [Sphinx] and hosted on [Read The Docs]. Versions of the documentation
+for each augur release and git branch are available and preserved.
 
-Building the documentation locally is useful to test changes.
-First, make sure you have the development dependencies of augur installed:
+Building the documentation locally is useful to test changes.First, make sure you have the development dependencies of augur installed:
```

Comment:
This isn't written down anywhere except in old PR comments, but @trvrb's preference for Markdown files (which we adhere to within augur) is put each sentence on its own line and to use inline link syntax instead of link reference syntax. The tips you've added here are great, but would be good to readjust the Markdown style to stay consistent.